speed-dreams/source-2.2.3/src/modules/track/trackv1/track4.cpp

```diff
@@ -1232,6 +1232,8 @@ CreateSegRing(void *TrackHandle, tTrack *theTrack, tTrackSeg *start, tTrackSeg *
     int		ind = 0;
 
     radius = arc = length = alf = xr = yr = newxr = newyr = xl = yl = 0;
+    // SIMULATED DRIVING ASSISTANCE: added speedLimit
+    float   speedLimit = -std::numeric_limits<float>::infinity();
```

Comment:
Pretty sure this gets overwritten by the read from the xml file, so can be removed